multi-node/libvirt/kube-master/openstack/latest/user_data

```diff
@@ -0,0 +1,616 @@
+#!/bin/bash
```

Comment:
Yeah, it handles installing TLS/SSH assets, finding the right IP address without a metadata service to query, and making etcd advertise the correct IP address.